core/src/test/scala/org/apache/spark/SortShuffleSuite.scala

```diff
@@ -83,8 +83,8 @@ class SortShuffleSuite extends ShuffleSuite with BeforeAndAfterAll {
     shuffledRdd.count()
     // Ensure that the shuffle actually created files that will need to be cleaned up
     val filesCreatedByShuffle = getAllFiles -- filesBeforeShuffle
-    filesCreatedByShuffle.map(_.getName) should be
-    Set("shuffle_0_0_0.data", "shuffle_0_0_0.index")
```

Comment:
this wasn't actually testing anything earlier.  It was getting parsed as two entirely different statements, due to some wrinkle in the way the `be` scalatest construct work.